src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/Program.cs

```diff
@@ -69,16 +79,136 @@ private void DoTheRun(int iterations)
             StaticDisposablesHelper.DisposeAll();
         }
 
-        public void RunAllStressTests()
+        public static void RunAllStressTests()
+        {
+            //A full cycle of create/use/destroy
+            CreateAndCloseFactoryAndChannelFullCycleTest<WcfService1.IService1, HelloWorldTest>.CreateFactoriesAndChannelsUseAllOnceCloseAll();
+            //CreateAndCloseFactoryAndChannelFullCycleTest<WcfService1.IDuplexService, DuplexTest>.CreateFactoriesAndChannelsUseAllOnceCloseAll();
+            CreateAndCloseFactoryAndChannelFullCycleTest<WcfService1.IStreamingService, StreamingTest>.CreateFactoriesAndChannelsUseAllOnceCloseAll();
+
+            // Pool channel factories and always create/destroy channels
+            PooledFactories<WcfService1.IService1, HelloWorldTest>.CreateUseAndCloseChannels();
+            //PooledFactories<WcfService1.IDuplexService, DuplexTest>.CreateUseAndCloseChannels();
+            PooledFactories<WcfService1.IStreamingService, StreamingTest>.CreateUseAndCloseChannels();
+
+            // Pool channel factories, always create/destroy channels, recycle factories while using them
+            RecyclablePooledFactories<WcfService1.IService1, HelloWorldTest>.RunAllScenariosWithWeights(100, 1);
+            //RecyclablePooledFactories<WcfService1.IDuplexService, DuplexTest>.RunAllScenariosWithWeights(100, 1);
+            RecyclablePooledFactories<WcfService1.IStreamingService, StreamingTest>.RunAllScenariosWithWeights(100, 1);
+
+            // Pool both channel factories and channels, never recycle
+            PooledFactoriesAndChannels<WcfService1.IService1, HelloWorldTest>.UseChannelsInPooledFactoriesAndChannels();
+            //PooledFactoriesAndChannels<WcfService1.IDuplexService, DuplexTest>.UseChannelsInPooledFactoriesAndChannels();
+            PooledFactoriesAndChannels<WcfService1.IStreamingService, StreamingTest>.UseChannelsInPooledFactoriesAndChannels();
+
+
+            //// These will get stuck with wcf/issues/108
```

Comment:
Can you please open an issue and link to #108 that this needs to be reactivated once 108 is fixed? Ditto with the Duplex cases that are currently commented out.